addons/website/static/src/snippets/s_social_media/options.js

```diff
@@ -0,0 +1,208 @@
+/** @odoo-module **/
+
+import options from 'web_editor.snippets.options';
+import {websiteDomain} from 'website.utils';
+
+options.registry.SocialMedia = options.Class.extend({
+    /**
+     * @override
+     */
+    async willStart() {
+        this._super(...arguments);
+        const websiteId = websiteDomain(this)[2][2];
+        const target = this.$target[0];
+        // Fetch URLs for db links.
+        const res = await this._rpc({
+            model: 'website',
+            method: 'search_read',
+            args: [[['id', '=', websiteId]], ['social_facebook', 'social_twitter', 'social_youtube',
+                'social_instagram', 'social_linkedin', 'social_github']],
+            limit: 1,
+        });
+        if (res) {
```

Comment:
res can't be falsy, right? useless condition imo
If so, you can also add [0] after `await this._rpc({ ... })` so `res` is actually already `res[0]` and you don't need to repeat res[0]. (or do `const [res] = await this._rpc({..`